src/sentry/testutils/factories.py

```diff
@@ -1011,3 +1015,39 @@ def create_codeowners(project, code_mapping, **kwargs):
         return ProjectCodeOwners.objects.create(
             project=project, repository_project_path_config=code_mapping, **kwargs
         )
+
+    @staticmethod
+    def create_integration(
+        organization: Organization, provider: ExternalProviders, external_id: str, **kwargs: Any
+    ) -> Identity:
```

Comment:
Is this right? It says you're returning an `Identity` but you are returning an `Integration`

Reply:
Also I don't know if it's too specific but I dont know that this would actually work for non-Slack integrations given the metadata, so you might want to just rename this `create_slack_integration`.